uber/site_sections/registration.py

```diff
@@ -923,7 +923,7 @@ def stats(self):
         return json.dumps({
             'badges_sold': c.BADGES_SOLD,
             'remaining_badges': c.REMAINING_BADGES,
-
+            'ticket_price': c.BADGE_PRICE,
```

Comment:
We named everything else 'badges' in these dumps, is there a particular reason to say "ticket" here?